File: app/src/main/java/com/callmangement/EHR/imagepicker/listener/OnImageLoaderListener.java
package com.callmangement.EHR.imagepicker.listener;



import com.callmangement.EHR.imagepicker.model.Folder;
import com.callmangement.EHR.imagepicker.model.Image;

import java.util.List;

/**
 * Created by hoanglam on 8/17/17.
 */

public interface OnImageLoaderListener {
    void onImageLoaded(List<Image> images, List<Folder> folders);

    void onFailed(Throwable throwable);
}

File: app/src/main/java/com/callmangement/ehr/imagepicker/listener/OnImageLoaderListener.kt
package com.callmangement.ehr.imagepicker.listener

import com.callmangement.ehr.imagepicker.model.Folder
import com.callmangement.ehr.imagepicker.model.Image


/**
 * Created by hoanglam on 8/17/17.
 */
interface OnImageLoaderListener {
    fun onImageLoaded(images: List<Image>?, folders: List<Folder>?)

    fun onFailed(throwable: Throwable?)
}
